### PR TITLE
minor javadoc tweak

### PR DESCRIPTION
**Pull request by bavardage**
per the signature, think this should be storeStream

### CODE REVIEW

**Comment by bavardage** [comment]
@rjullman

File: atlasdb-client/src/main/java/com/palantir/atlasdb/stream/PersistentStreamStore.java [hunk 1]
```diff
@@ -82,7 +82,7 @@
      * Although this method provides performance gains by batching the relevant underlying code it can also also
      * cause the underlying transaction to be long-running.  When possible, individually add streams using
      * {@link #getByHashOrStoreStreamAndMarkAsUsed(Transaction, Sha256Hash, InputStream, byte[])} or
-     * {@line #storeStreams(InputStream)}.
+     * {@line #storeStream(InputStream)}.
```
Comment (jared2501) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/stream/PersistentStreamStore.java]: `{@link #storeStream(InputStream)}`

Comment (bavardage) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/stream/PersistentStreamStore.java]: good catch, fixed

**Comment by bavardage** [comment]
is there a list to ping for PRs?
@schlosna ?

**Pull request merged**